src/Core/Constants.cs

```diff
@@ -96,12 +96,17 @@ public static class FeatureFlagKeys
     public const string VaultOnboarding = "vault-onboarding";
     public const string AutofillV2 = "autofill-v2";
     public const string BrowserFilelessImport = "browser-fileless-import";
-    public const string FlexibleCollections = "flexible-collections";
+    /// <summary>
+    /// Deprecated - never used, do not use. Will always default to false. Will be deleted as part of Flexible Collections cleanup
+    /// </summary>
+    public const string FlexibleCollections = "flexible-collections-disabled-do-not-use";
```

Comment:
This is now only being used in the repository calls to use the `v2` sprocs, which don't contain AccessAll logic. However, we're now relying on setting AccessAll to `0`, rather than changing out the sprocs. This can always default to false to use the existing sprocs.

For safety, I intentionally broke the key here so that it can't be turned on. Really it should be deleted but I didn't want to trigger additional codeowner review - I'll clean up the remaining references in https://bitwarden.atlassian.net/browse/AC-2061